Remotion/Data/DomainObjects/Persistence/Rdbms/SqlServer/IDateTimeDefaultStorageTypeProvider.cs

```diff
@@ -0,0 +1,10 @@
+﻿// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System.Data;
+
+namespace Remotion.Data.DomainObjects.Persistence.Rdbms.SqlServer;
+
+public interface IDateTimeDefaultStorageTypeProvider
```

Comment:
One alternative on the design, given how it is now done with two types; just make two properties directly on the interface one for the storagetypename and one for dbtype.

The original idea with the result object was...design conincidence I gess